src/optimalcontrol/src/Controller.cpp

```diff
@@ -22,7 +22,7 @@ namespace iDynTree {
     Controller::~Controller()
     {}
 
-    bool Controller::setStateFeedback(const double t, const VectorDynSize &stateFeedback){
```

Comment:
There is any reason why you removed the `const` here?

Reply:
There was not a ``const`` there, was it?

Reply:
well.. you removed it, so I assumed it was there.
Anyway, on the primitive type is makes no difference

Reply:
Sorry, for some reason I was searching for an hypothetical ``const`` after the method definition. Yes,   it was not necessary there.